src/components/ScreenSplit.jsx

```diff
@@ -5,48 +5,46 @@ import { makeStyles } from "@material-ui/core/styles";
 const useStyles = makeStyles((theme) => ({
   gridItem: {
     [theme.breakpoints.up("xs")]: {
-      marginTop: "20%",
-      marginBottom: "20%"
+      marginBottom: "-130%",
+      height: "80vh",
     },
     [theme.breakpoints.up("sm")]: {
-      marginTop: "40%",
-      marginBottom: "20%"
+      marginTop: "0",
+      marginBottom: "0",
     },
     [theme.breakpoints.up("lg")]: {
-      marginTop: "0%",
-      marginBottom: "0%"
+      marginTop: "0",
+      marginBottom: "0",
+      height: "80vh",
+      display: "flex",
+      flexDirection: "column",
+      padding: "10px",
     },
   },
 }));
 
-const ScreenSplit = ({ left=(<></>), right=(<></>), centered = false }) => {
-  const classes = useStyles()
-  let styleGridItem = {
-    height: "80vh",
-    display: "flex",
-    flexDirection: "column",
-    padding: "10px",
-  }
+const ScreenSplit = ({ left = <></>, right = <></>, centered = false }) => {
+  const classes = useStyles();
 
   if (centered) {
-    styleGridItem = {
-      ...styleGridItem,
+    classes.GridItem = {
+      ...classes.styleGridItem,
       justifyContent: "center",
     };
   }
 
   return (
-    <Grid container>
+    <Grid container className={classes.GridItem}>
```

Comment:
Not sure, but it is a grid container, should it have the same class as item?

Reply:
the thing is that it is used on 2 other boxes (items in the grid) so i thought grid.. but i saw a different mistake i made here... used pascal somehow instead of camel..